geospaas_harvesting/ingesters.py

```diff
@@ -510,6 +513,67 @@ def _get_normalized_attributes(self, dataset_info, *args, **kwargs):
         return normalized_attributes
 
 
+class NetCDFIngester(MetanormIngester):
+    """Ingests metadata from NetCDF files. The files can be either
+    local or remote (if the remote repository supports it).
+    """
+
+    def _get_geometry_wkt(self, dataset):
+        """Extracts the WKT string for the dataset's spatial coverage
+        """
+        raise NotImplementedError()
+
+    def _get_raw_attributes(self, dataset_path):
+        """Get the raw metadata from the NetCDF file"""
+        dataset = netCDF4.Dataset(dataset_path)
+        raw_attributes = dataset.__dict__
+        self.add_url(dataset_path, raw_attributes)
+        raw_attributes['geometry'] = self._get_geometry_wkt(dataset)
+        raw_attributes['raw_dataset_parameters'] = self._get_parameter_names(dataset)
+        return raw_attributes
+
+    def _get_parameter_names(self, dataset):
+        """Get the names of the dataset's variables"""
+        return [
+            variable.standard_name
+            for variable in dataset.variables.values()
+            if hasattr(variable, 'standard_name')
+        ]
+
+    def _get_normalized_attributes(self, dataset_info, *args, **kwargs):
+        raw_attributes = self._get_raw_attributes(dataset_info)
+        return self._metadata_handler.get_parameters(raw_attributes)
+
+
+class OneDimensionNetCDFIngester(NetCDFIngester):
+    """NetCDF ingester able to extract one-dimensional geographic
+    locations"""
+
+    def _get_geometry_wkt(self, dataset):
+        longitudes = dataset.variables['LONGITUDE']
+        latitudes = dataset.variables['LATITUDE']
```

Comment:
there is no dataset.variables['LONGITUDE'] in jaxa case:
![image](https://user-images.githubusercontent.com/62345618/107346953-98d30580-6ac5-11eb-9f9d-16eb0a592919.png)

Reply:
If you are talking about AMSR2 data, it is not one-dimensional so it should not be read using the `OneDimensionNetCDFIngester` class.